database/migrations/2020_12_24_053624_add_status_to_rooms_table.php

```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
-class AddStatusToRooms extends Migration
+class AddStatusToRoomsTable extends Migration
```

Comment:
Normally this is a bad practice, since its name is used to determine if it has already been run. 

However, since we are still pre-deployment I'll allow it

Reply:
Also bad practice to not follow a naming convention 🤷